Resources/public/js/models/ez-locationmodel.js

```diff
@@ -98,7 +98,27 @@ YUI.add('ez-locationmodel', function (Y) {
          */
         move: function (options, parentLocationId, callback) {
             options.api.getContentService().moveSubtree(this.get('id'), parentLocationId, callback);
-        }
+        },
+
+        //TODO comment
+        _updatehidden: function (options, hidden, callback) {
```

Comment:
we keep it like that to be consistant with naming used in the API. We will of course not use these terms in error messages and such :)